crates/timestamp/Cargo.toml

```
@@ -0,0 +1,43 @@
+[package]
+name = "holochain_timestamp"
+version = "0.5.0-dev.0"
+description = "Microsecond-precision timestamp datatype for Holochain"
+license = "Apache-2.0"
+homepage = "https://github.com/holochain/holochain"
+documentation = "https://docs.rs/holochain_timestamp"
+authors = ["Holochain Core Dev Team <devcore@holochain.org>"]
+keywords = ["holochain", "p2p", "dht", "networking"]
+categories = ["network-programming"]
```

[date-and-time](https://crates.io/category_slugs) maybe?

```suggestion
categories = ["date-and-time"]
```

---

Yes, good point!